test/helpers/testabilityPatch.js

```diff
@@ -37,6 +37,13 @@ beforeEach(function() {
 afterEach(function() {
   var count, cache;
 
+  // both of these nodes are presisted across tests
```

Comment:
It causes a collision to happen since the counter for the HashMap is reset between tests, so you may end up having the same hashCode for two different elements since the old one was stored on the HTML or BODY tag from a previous test.